### ISSUE TITLE
individual label and board colors

### ISSUE THREAD

**Issue by sonntagd** [wekan/wekan#457]
I would like to have more than the given ten colors for labels. They should probably be added with a color picker. For most use cases the given color set is enough, but I have at least one use case where I need more colors for labels.

Six colors for boards is also too little.

**Comment by augustogiles** [comment]
@choclin, its easy hard code add more colors.
Just edit the files in client/components/cards/label.styl
and add the colors in ASCII that u want;
 in models/board.js (exactly in labels.$.color) add the new names in 
allowedValues array. I edited my local wekan to 20 label colors.
For board probably is the same in other file.

**Comment by tobleron** [comment]
@augustogiles  I could not find this path in my wekan installation:
client/components/cards/label.styl

Could you please clarify where is it?

**Comment by xet7** [comment]
@tobleron 

It's labels.styl:
https://github.com/wekan/wekan/blob/devel/client/components/cards/labels.styl

In Wekan installations, built javascript could have different directory structure.

**Comment by JamesLavin** [comment]
@xet7 Thank you for building Wekan! I went looking for something like it yesterday, found it, installed it (via Docker testing), and am loving it.

I also hit this issue (insufficient number of tag colors) and tried the suggested solution above but realized I can't modify the Docker image. So I installed Wekan on Ubuntu using a Snap. Then I tried adding colors to the file you mentioned only to discover that Ubuntu snap files are installed as a read-only file system (https://askubuntu.com/questions/919091/why-snap-files-cannot-be-modified-in-any-way), so this solution doesn't work for either of the ways I've installed Wekan.

It would be really nice if more colors were available by default, and I suspect adding more colors wouldn't take up much time. Heck, I'll try doing so now. If I succeed, I'll submit a PR.
